ci/testgrid/config.yaml

```diff
@@ -135,6 +135,9 @@ dashboards:
     test_group_name: ci-knative-serving-api-coverage
     description: 'Conformance tests API coverage.'
     base_options: 'exclude-filter-by-regex=Overall$&group-by-directory=&expand-groups=&sort-by-name='
+  - name: conformance
```

Comment:
should we call this conformance-tests?

Reply:
I don't have a preference, I will wait for others to comment to make sure there is a consensus, before making a change.

Reply:
conformance-tests sounds good to me

Reply:
and are we taking conformance out of existing "continuous" tab?

Reply:
This PR doesn't do that, but can be done. My sense was that continuous would display results of all the tests that were run as part of continuous. Would you prefer taking them out with an exclude clause?

Reply:
I prefer not removing them from the CI tab. That's the canonical way of checking that everything is green, and our "green" concept is based on looking at all the tests in it.

Reply:
Changed tab to "conformance-tests"